.evg.yml

```diff
@@ -1,6 +1,18 @@
 stepback: true
 command_type: system
 
+variables:
+  - &assume_role_cmd
+    command: ec2.assume_role
+    params:
+      role_arn: ${assume_role_arn}
+      duration_seconds: 3600
+  - &aws_std_config
```

Comment:
If we want to reuse this in our main branch when we merge, we should use a more meaningful name.
Maybe something like `mongo-jdbc-driver-eap_bucket_config` or `evg_bucket_config`? 
I think that having the bucket name as part of the variable name makes it easier to know where we are writing to. Our release bucket or the project specific bucket for example.

Is this bucket a release bucket or a "temp" evg bucket where we put the result of tasks that we want to capture but will only keep for about 60 days (I think that's how much time the data is kept, I might be off)?
Right now, it seems like you expect it to be a temp bucket.

Reply:
I was thinking it would be the temp bucket, since the eap branch doesn't have access to `evg-mongo-jdbc-driver` nor `mciuploads`, and given we dont have any ticketing for updating the release/publishing of the eap version.

Reply:
This ticket is for publishing the EAP version to S3 because the publishing is done via Maven for the master branch, so I am not sure what Nathan had in mind. We'll ask him when he is back.
I wonder if we are Ok "loosing" the releases after a while or not 🤔 
I'd be happy to keep them around if possible, but if the delay is indeed 60 days, then it should be more than enough.